app/controllers/events_controller.rb

```diff
@@ -125,4 +132,16 @@ def generate_calendar
 
     calendar
   end
+
+  def send_new_event_email(event)
```

Comment:
I would rather see this as a method on `Event` where you could call `event.mail_users` in the method above instead of in the controller. Or at least moved to a private method as this is not an action that can be called by users.